src/logic/map_objects/tribes/building_settings.cc

```diff
@@ -145,7 +145,7 @@ constexpr uint8_t kCurrentPacketVersion = 2;
 constexpr uint8_t kCurrentPacketVersionMilitarysite = 1;
 constexpr uint8_t kCurrentPacketVersionProductionsite = 2;
 constexpr uint8_t kCurrentPacketVersionTrainingsite = 1;
-constexpr uint8_t kCurrentPacketVersionWarehouse = 2;
+constexpr uint8_t kCurrentPacketVersionWarehouseAndExpedition = 2;
```

Comment:
Why did you rename this? This is only for building _settings_, no expedition-related stuff here.

Reply:
Because of 
```
	// Keep this synchronized with kCurrentPacketVersionWarehouse in MapBuildingDataPacket!!
	static const uint16_t kCurrentPacketVersion = 8;
```

So it gives people a hint about where it is being used - when loading a warehouse and when loading an expedition.

Reply:
That hint refers to the relation between a Warehouse and an ExpeditionBootstrap. But this file is for BuildingSettings which are used only by ConstructionSites. These are neither synchronized with the warehouse/expedition packets nor meant to be.

Reply:
Duh, now I get it. My brain was in the new header file.